hw/drivers/trng/trng_da1469x/include/trng_da1469x.h

```diff
@@ -27,6 +27,9 @@ extern "C" {
 #endif
 
 int da1469x_trng_init(struct os_dev *dev, void *arg);
+#ifdef MBEDTLS_ENTROPY_HARDWARE_ALT
+int mbedtls_hardware_poll( void *data, unsigned char *output, size_t len, size_t *olen );
```

Comment:
this prototype actually exists in entorpy_poll.h so placing it here again seems strange

Reply:
I'll check on that